src/App.js

```diff
@@ -29,7 +25,19 @@ const [message, setMessage] = useState('');
   return (
     <>
       <h1>{message}</h1>
-      <Contact />
+      <Routes>
+        <Route path="/register" element={<UserRegistration />}></Route>
+        <Route path="/signIn" element={<Contact />}></Route>
+        {/* only logged in user can visit home page */}
+        <Route
+          path="/home"
+          element={
+            <RequireAuth>
+              <Home />
+            </RequireAuth>
+          }
+        ></Route>
+      </Routes>
```

Comment:
Looks great!

Right now, if none of the routes are matched (so for example for localhost:8000, it won't match `/register`, `/signIn`, or `/home`) it will have a warning in the console:
```
No routes matched location "/" 
````

and it will just render whatever else this component has, which right now is just the message. We might want to default redirect users to the appropriate page as soon as the load the root url. Eg, we could base it on whether they are logged in or not:

```js
import { Routes, Route, Navigate } from 'react-router-dom';
import { AuthStatus, useAuth } from '@akosasante/react-auth-context';

function App() {
  const [message, setMessage] = useState('');
  const { status } = useAuth();
  const isLoggedIn = status === AuthStatus.LoggedIn;

  return (
    <>
      <h1>{message}</h1>
      <Routes>
        <Route path="/" element={isLoggedIn ? <Home /> : <Navigate to="/signIn" /> }></Route>
        <Route path="/register" element={<UserRegistration />}></Route>
        <Route path="/signIn" element={<Contact />}></Route>
        <Route path="/home" element={<Home />}></Route>
      </Routes>
    </>
  );
}
```